react-app-drawer-component/src/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { GiHamburgerMenu } from 'react-icons/gi';
+
+class Drawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { menuOpen: false };
+    this.handleClick = this.handleClick.bind(this);
+    this.handleTabClick = this.handleTabClick.bind(this);
+    this.container = 'container';
```

Comment:
It's not good to store extra stuff on `this` if you can avoid it. Try computing your class name in your render method instead